seed/checkUserExistence.js

```diff
@@ -0,0 +1,25 @@
+const User = require('~/models/user')
+const Role = require('~/models/role')
+const {
+  roles: { SUPERADMIN }
+} = require('~/consts/auth')
+const SeedSuperAdmin = require('~/seed/seedSuperAdmin')
+const checkRoleExistence = require('~/seed/checkRoleExistence')
+const logger = require('~/logger/logger')
+
+const checkUserExistence = async () => {
```

Comment:
Same here regarding naming - you don't just check for user existence.

Reply:
answer is in the comment above